release-team/role-handbooks/communications/README.md

```diff
@@ -52,6 +52,7 @@ There is a channel on the Kubernetes Slack workspace, `release-comms`, which is
 
 - Begin analyzing the release notes draft
 - Work with the enhancements lead to understand what might be delivered
+- Establish initial meeting with team to introduce everyone, collective tasks, and upcoming timelines
```

Comment:
Why in the week 3? And not earlier.

Reply:
because shadows tend to be finalized in Week 2, depending

I take your point, though - earlier could be better!

Reply:
Ah! Ok makes sense then. Thanks for the clarification @guineveresaenger